app/views/spree/checkout/payment/_afterpay.html.erb

```diff
@@ -5,4 +5,5 @@
   class="hidden"
   data-payment-method-id="<%= payment_method.id %>"
   data-order-number="<%= @order.number %>"
+  data-popup-window="<%= payment_method.preferred_popup_window%>"
```

Comment:
Watch out! this line is missing a space at the end.

Reply:
Done!